cmd/imagec/errors.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+// DoNotRetry is an error wrapper indicating that the error cannot be resolved with a retry.
+type DoNotRetry struct {
+	Err error
```

Comment:
Well normally I would not have users of the package create errors that are in the package. They are only exported so that callers can type assert or inspect the error. In this case if you want callers to get to the actual underlying error then yes you can't just embed error and what you have is appropriate. I don't see that is your PR right now but perhaps it would be useful in the future.